pkg/cmd/dev/dev.go

```diff
@@ -132,10 +135,7 @@ Typical usage:
 
 	// Add all the shared flags.
 	var debugVar bool
-	var bazelcmd string
 	ret.cli.PersistentFlags().BoolVar(&debugVar, "debug", false, "enable debug logging for dev")
-	ret.cli.PersistentFlags().StringVar(&bazelcmd, "", "",
```

Comment:
@irfansharif ditto. was this intentional?

Reply:
Yup, they're moved to the top level help text. Before it looked like this which I found confusing:

```
Flags:
      -- string   pass additional arguments directly to bazel; e.g. '-- --sandbox_debug'.
      --debug     enable debug logging for dev
  -h, --help      help for dev
  -v, --version   version for dev
```

Now it looks like this:

```
Typical usage:
...
    dev build short -- --verbose_failures --profile=prof.gz
                Pass additional arguments directly to bazel.
```